nupic/research/frameworks/vernon/mixins/rezero_weights.py

```diff
@@ -49,7 +49,7 @@ def create_model(cls, config, device):
         # Some initialization strategies can destroy sparsity, so we call rezero
         # here.
         for module in model.modules():
-            if isinstance(module, SparseWeightsBase):
+            if isinstance(module, HasRezeroWeights):
```

Comment:
(minor comment) I see you made this same change in #416, any reason you're re-doing it here?

Reply:
When I made this PR, the other one (#416) hadn't been merged yet, but I built off of its changes. You'll see that once this PR is up to date with the master branch that these diffs won't be highlighted anymore.